include/nuttx/signal.h

```diff
@@ -191,6 +204,99 @@ int nxsig_addset(FAR sigset_t *set, int signo);
 
 int nxsig_delset(FAR sigset_t *set, int signo);
 
+/****************************************************************************
+ * Name: nxsig_addmultiple
+ *
+ * Description:
+ *   This function adds multiple signals specified by addset to the signal
+ *   set specified by source.  The result is returned in dest.
+ *
+ * Input Parameters:
+ *   dest   - The location to return the result
+ *   source - Signal set to add signal to
+ *   addset - Signals to add
+ *
+ * Returned Value:
+ *   This is an internal OS interface and should not be used by applications.
+ *   It follows the NuttX internal error return policy:  Zero (OK) is
+ *   returned on success.  A negated errno value is returned on failure.
+ *
+ * Assumptions:
+ *
+ ****************************************************************************/
+
+int nxsig_addmultiple(FAR sigset_t *dest, FAR const sigset_t *source,
+                      FAR const sigset_t *addset);
+
+/****************************************************************************
+ * Name: nxsig_delmultiple
+ *
+ * Description:
+ *   This function deletes multiple signals specified by delset from the
+ *   signal set specified by the 'source' argument.  The result is returned
+ *   in the dest argument.
+ *
+ * Input Parameters:
+ *   dest   - The location to return the result
+ *   source - The set to delete the signal from
+ *   delset - Signals to delete
+ *
+ * Returned Value:
+ *   This is an internal OS interface and should not be used by applications.
+ *   It follows the NuttX internal error return policy:  Zero (OK) is
+ *   returned on success.  A negated errno value is returned on failure.
+ *
+ * Assumptions:
+ *
+ ****************************************************************************/
+
+int nxsig_delmultiple(FAR sigset_t *dest, FAR const sigset_t *source,
+                      FAR const sigset_t *delset);
+
+/****************************************************************************
+ * Name: nxsig_complement
+ *
+ * Description:
+ *   This function comples the signal set specified by source and places the
+ *   result in dest.
+ *
+ * Input Parameters:
+ *   dest - The location to return the result
+ *   source - The signal set to complement
+ *
+ * Returned Value:
+ *   0 (OK), or -1 (ERROR) if any error occurs.
+ *
+ * Assumptions:
+ *
+ ****************************************************************************/
+
+int nxsig_complement(FAR sigset_t *dest, FAR const sigset_t *source);
```

Comment:
Seems like this is unused

Reply:
Yes, it is not used.  However, it is in a library and will not be included in the build.  I thought it could be useful in the future.  I will remove it.

Reply:
Removed